modules/@apostrophecms/schema/index.js

```diff
@@ -94,17 +94,6 @@ module.exports = {
       }
     });
 
-    self.addFieldType({
```

Comment:
Unrelated, but I think I ran across some error message that led me here.

Reply:
LOL